pkg/rtc/transport.go

```diff
@@ -1951,6 +2000,10 @@ func (t *PCTransport) handleRemoteOfferReceived(sd *webrtc.SessionDescription) e
 		t.resetShortConn()
 	}
 
+	if offerRestartICE {
+		t.outputAndClearICEStats()
```

Comment:
we have a lot ICE restart due to signalling channel disconnect. Wondering if this will be logging a lot. Probably can keep an eye and tune later.

Reply:
It will be ok that the stas only collected one second before failed timeout, so a normal restart will not produce logs. Output here because the restart will clear ice agent's state so we have no chance to get it after restart

Reply:
Got it.